be/src/runtime/runtime_filter_worker.h

```diff
@@ -120,9 +120,13 @@ class RuntimeFilterWorker {
     void execute();
     void send_part_runtime_filter(PTransmitRuntimeFilterParams&& params,
                                   const std::vector<starrocks::TNetworkAddress>& addrs, int timeout_ms);
+    void send_broadcast_runtime_filter(PTransmitRuntimeFilterParams&& params,
```

Comment:
`send_broadcast_runtime_filter` and `_send_broadcast_runtime_filter` have same method names and same method args, which is confusing.

Reply:
the latter is renamed to _process_send_broadcast_runtime_filter_event?